tests/test_bukuDb.py

```diff
@@ -1527,30 +1523,32 @@ def test_browse_by_index(low, high, index, is_range, empty_database):
     with mock.patch("buku.browse"):
         import buku
 
-        bdb = buku.BukuDb()
-        bdb.delete_rec_all()
-        db_len = 0
-        if not empty_database:
-            bdb.add_rec("https://www.google.com/ncr", "?")
-            db_len += 1
-        res = bdb.browse_by_index(index=index, low=low, high=high, is_range=is_range)
-        if is_range and (low < 0 or high < 0):
-            assert not res
-        elif is_range and n_low > 0 and n_high > 0:
-            assert res
-        elif is_range:
-            assert not res
-        elif not is_range and index < 0:
-            assert not res
-        elif not is_range and index > db_len:
-            assert not res
-        elif not is_range and index >= 0 and empty_database:
-            assert not res
-        elif not is_range and 0 <= index <= db_len and not empty_database:
-            assert res
-        else:
-            raise ValueError
-        bdb.delete_rec_all()
+        bdb = buku.BukuDb(TEST_TEMP_DBFILE_PATH)
+        try:
+            bdb.delete_rec_all()
+            db_len = 0
+            if not empty_database:
+                bdb.add_rec("https://www.google.com/ncr", "?")
+                db_len += 1
+            res = bdb.browse_by_index(index=index, low=low, high=high, is_range=is_range)
+            if is_range and (low < 0 or high < 0):
+                assert not res
+            elif is_range and n_low > 0 and n_high > 0:
+                assert res
+            elif is_range:
+                assert not res
+            elif not is_range and index < 0:
+                assert not res
+            elif not is_range and index > db_len:
+                assert not res
+            elif not is_range and index >= 0 and empty_database:
+                assert not res
+            elif not is_range and 0 <= index <= db_len and not empty_database:
+                assert res
+            else:
+                raise ValueError
+        finally:
+            rmdb(bdb)
```

Comment:
Here I'm fixing some testing regressions on Windows (or rather, errors that did not happen to pop up before some of the more recent tests got added)